src/java.base/share/classes/java/lang/classfile/TypeKind.java

```diff
@@ -32,6 +32,8 @@
 import jdk.internal.javac.PreviewFeature;
 import jdk.internal.vm.annotation.Stable;
 
+import static java.lang.constant.ConstantDescs.*;
```

Comment:
Maybe import the primitive class descriptors from `PrimitiveClassDescImpl` in order to avoid unnecessary initialisation of unrelated `ConstantDesc`s:
```suggestion
import static jdk.internal.constant.PrimitiveClassDescImpl.*;
```